Mathlib/Topology/Algebra/ContinuousMonoidHom.lean

```diff
@@ -376,4 +379,75 @@ def compRight {B : Type*} [CommGroup B] [TopologicalSpace B] [TopologicalGroup B
 #align continuous_monoid_hom.comp_right ContinuousMonoidHom.compRight
 #align continuous_add_monoid_hom.comp_right ContinuousAddMonoidHom.compRight
 
+section LocallyCompact
+
+variable {X Y : Type*} [TopologicalSpace X] [Group X] [TopologicalGroup X]
+  [UniformSpace Y] [CommGroup Y] [UniformGroup Y] [T0Space Y] [CompactSpace Y]
+
+theorem locallyCompactSpace_of_equicontinuousAt (U : Set X) (V : Set Y)
```

Comment:
Is there a reason this doesn't work? (I mean, I understand it's not really associated to the Pontryagin Dual, but :shrug:)
```suggestion
@[to_additive]
theorem locallyCompactSpace_of_equicontinuousAt (U : Set X) (V : Set Y)
```